llama_stack/distribution/server/server.py

```diff
@@ -278,8 +281,20 @@ def main(
         config = StackRunConfig(**yaml.safe_load(fp))
 
     app = FastAPI()
+    # instantiate kvstore for storing and retrieving distribution metadata
+    dist_kvstore = asyncio.run(
```

Comment:
could we allow this to be configured? by adding an optional key in `StackRunConfig` just like agent has `persistence_store`?